src/sender.rs

```diff
@@ -1036,7 +1037,7 @@ where
                     } = sent;
                     UnidentifiedDeliveryStatus {
                         destination_service_id: Some(
-                            recipient.uuid.to_string(),
+                            recipient.service_id_string(),
```

Comment:
Is this double checked with upstream?

Reply:
The name of the field _really_ suggests that it should be like that.

Reply:
Good enough!